jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialFileHelpNotifications.kt

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.credentials
+
+import com.intellij.ide.util.PropertiesComponent
+import com.intellij.openapi.fileEditor.FileEditor
+import com.intellij.openapi.help.HelpManager
+import com.intellij.openapi.project.DumbAware
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.util.Key
+import com.intellij.openapi.vfs.VirtualFile
+import com.intellij.ui.EditorNotificationPanel
+import com.intellij.ui.EditorNotifications
+import software.amazon.awssdk.profiles.ProfileFileLocation
+import software.aws.toolkits.jetbrains.core.credentials.CredentialFileHelpNotifications.CredentialFileNotificationPanel
+import software.aws.toolkits.jetbrains.core.help.HelpIds
+import software.aws.toolkits.resources.message
+
+class CredentialFileHelpNotifications : EditorNotifications.Provider<CredentialFileNotificationPanel>(), DumbAware {
+    override fun getKey(): Key<CredentialFileNotificationPanel> = KEY
+
+    override fun createNotificationPanel(file: VirtualFile, fileEditor: FileEditor, project: Project): CredentialFileNotificationPanel? {
+        if (PropertiesComponent.getInstance().isTrueValue(DISABLE_KEY)) return null
```

Comment:
makes sense if you read the rest of the file but probably worth a comment explaining these conditions

Reply:
Added